### ISSUE TITLE
Zero_loss function for EBGAN

### ISSUE THREAD

**Issue by adnan1306** [tatsy/keras-generative#12]
Can you please explain how the zero loss function exactly works?
def zero_loss(y_true, y_pred):
    return K.zeros_like(y_true)

Isn't the loss always zero if we use this function?

**Comment by tatsy** [comment]
You are right.
For Keras, we have to give the loss for each batch. The zero_loss function returns single zeros for each batch.
